fluster/decoders/gstreamer.py

```diff
@@ -705,12 +705,22 @@ class GStreamerMPEG2VideoDecoder(GStreamer10Video):
     api = "SW"
 
 
+@register_decoder
+class GStreamerMPEG4VideoDecoder(GStreamer10Video):
```

Comment:
This is the libav decoder gstreamer, let's mark it as such following previous implementations as examples, see https://github.com/fluendo/fluster/pull/273/files#diff-a9d5a9611f0ac07816284d84793cec8c5ad5086db1b25d0274d6172814325a10R223

```suggestion
class GStreamerLibavMPEG4VideoDecoder(GStreamer10Video):
```